ui/app/styles/components/allocation-fs.scss

```diff
@@ -0,0 +1,70 @@
+.fs-explorer.table {
+  width: 100%;
+
+  tbody {
+    a {
+      text-decoration: none;
+      color: inherit;
+
+      &:hover {
+        .name {
+          text-decoration: underline;
+        }
+      }
+    }
+  }
+}
+
+.breadcrumb.fs {
```

Comment:
Hmm. This is a weird one. I've been trying to keep everything in a component css file scoped to the one class that targets the component. So in this case something, I expect `.breadcrumbs` to be nested under `.fs-explorer`. 

What makes it weird is that this breadcrumbs pattern doesn't necessarily need to be tied to the fs-explorer. 

So, to conclude my thinking out loud, I think this needs to change in one of two ways.

1) `.breadcrumb.fs` gets nested under `.fs-explorer` so it can't be used outside of the `.fs-explorer` context.
2) `.breadcrumb.fs` gets its own top-level component class and file (e.g., `.secondary-breadcrumbs`) that is then composed as a part of the `fs-explorer`.

Reply:
👍🏻 I chose nesting, if it ends up being useful elsewhere, it would be easy to extract